### PR TITLE
docs: 添加tooltip destroyTooltipOnHide Api说明

### LINKED ISSUE #18833: Tooltip组件antd文档中没有 destroyTooltipOnHide Api说明
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
在ant-design文档中Tooltip组件destroyTooltipOnHide Api没有说明,开发者在通过antd文档开发时不知道有destroyTooltipOnHide Api
在rc-tooltip中是有这个Api的：
destroyTooltipOnHide | boolean | false | whether destroy tooltip when tooltip is hidden

### What does the proposed API look like?
希望在antd文档中添加上destroyTooltipOnHide Api说明

### PR DESCRIPTION
**Pull request by ooolog**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution
close #18833

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

-----
[View rendered components/tooltip/index.en-US.md](https://github.com/ooolog/ant-design/blob/fix-tooltip-apiprops/components/tooltip/index.en-US.md)
[View rendered components/tooltip/index.zh-CN.md](https://github.com/ooolog/ant-design/blob/fix-tooltip-apiprops/components/tooltip/index.zh-CN.md)

### CODE REVIEW

**Review by yoyo837 — CHANGES_REQUESTED** [review]
列头内容 错位了

**Comment by ooolog** [comment]
> 列头内容 错位了

😯，的确错误了，修复了

File: components/tooltip/index.zh-CN.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ title: Tooltip
 | trigger | 触发行为，可选 `hover/focus/click/contextMenu` | string | hover |  |
 | visible | 用于手动控制浮层显隐 | boolean | false |  |
 | onVisibleChange | 显示隐藏的回调 | (visible) => void | 无 |  |
+| destroyTooltipOnHide | 是否在隐藏工具提示时销毁工具提示 | boolean | false |  |
```
Comment (yoyo837) [inline components/tooltip/index.zh-CN.md]: > 销毁工具提示

这里的`工具提示` 应该不需要翻译

Comment (ooolog) [reply components/tooltip/index.zh-CN.md]: 空着就可以吗

Comment (ooolog) [reply components/tooltip/index.zh-CN.md]: ok了

**Comment by zombieJ** [comment]
这个官方文档里先不透出了，antd 里的命名规范需要改成 `destroyOnHide`。
ref #16048

**Pull request merged**